src/Webnative.elm

```diff
@@ -115,18 +115,31 @@ type alias AppPermissions =
     import Webnative.Path as Path
 
     { private =
-        [ Path.forPermissions (Path.directory [ "Audio", "Mixtapes" ])
-        , Path.forPermissions (Path.directory [ "Audio", "Playlists" ])
-        ]
+        { directories = [ Path.directory [ "Audio", "Mixtapes" ] ]
+        , files = [ Path.directory [ "Audio", "Playlists", "Jazz" ] ]
```

Comment:
This should be `Path.file`, right? And maybe something like `Jazz.json`?

Reply:
Yup!